docs/modules/ROOT/pages/index.adoc

```diff
@@ -47,6 +47,151 @@ If you want to use the Bundle generator, you will first need to use Quarkus 2.3.
+In the previous step, we generated the bundle manifests at `target/bundle` which includes a ready-to-use Dockerfile at `target/bundle/bundle.Dockerfile` file that you will use to build and push the final Operator Bundle image to your container registry:
+
+[source,shell script]
+----
+MY_BUNDLE_IMAGE=<your container registry>/<your container registry namespace>/<bundle image name>:<tag>
+docker build -t $MY_BUNDLE_IMAGE -f target/bundle/bundle.Dockerfile target/bundle
+docker push $MY_BUNDLE_IMAGE
+----
+
+For example, if we want to name our bundle image as `my-manifest-bundle`, our container registry is `quay.io`, our Quay user is `myuser` and the tag we're releasing is `1.0`, the final `MY_BUNDLE_IMAGE` property would be `quay.io/myuser/my-manifest-bundle:1.0`.
+
+3. Make your operator available within a Catalog
+
+OLM uses catalogs to discover and install Operators and their dependencies. So, a catalog is similar to a repository of operators and their associated versions that can be installed on a cluster. Moreover, the catalog is also a container image that contains a collection of bundles and channels. Therefore, we'd need to create a new catalog (or update an existing one if you're already have one), build/push the catalog image and then install it on our cluster.
+
+So far, we have already built the Operator bundle image at `$MY_BUNDLE_IMAGE` (see above) and next, we need to add this Operator bundle image into our catalog. For doing this, we'll use the `olm` tool as follows:
+
+[source,shell script]
+----
+CATALOG_IMAGE=<catalog container registry>/<catalog container registry namespace>/<catalog name>:<tag>
+opm index add \
+    --bundles $MY_BUNDLE_IMAGE \
+    --tag $CATALOG_IMAGE \
+    --build-tool docker
+docker push $CATALOG_IMAGE
+----
+
+For example, if our catalog name is `my-catalog`, our container registry for the catalog is `quay.io`, our Quay user is `myuser` and the container tag we're releasing is `59.0`, the final `CATALOG_IMAGE` property would be `quay.io/myuser/my-catalog:59.0`.
+
+[NOTE]
+====
+If you're using an insecure registry, you'd need to append the argument `--skip-tls` to the `opm index` command.
+====
+
+Once we have our catalog image built and pushed at `$CATALOG_IMAGE` image, we need to install it on our cluster using the `CatalogSource` resource by doing the next command:
+
+[source,shell script]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-catalog-source
+  namespace: operators
+spec:
+  sourceType: grpc
+  image: $CATALOG_IMAGE
+EOF
+----
+
+[NOTE]
+====
+We need to install the catalog source at the Kubernetes namespace where we installed OLM, which by default is `operators`.
+====
+
+Once the catalog is installed, you should see the catalog pod up and running:
+
+[source,shell script]
+----
+kubectl get pods -n <K8S NAMESPACE> --selector=olm.catalogSource=my-catalog-source
```

Comment:
I assume that `K8S NAMESPACE` refers to the namespace in which we created the `CatalogSource`, right? In this case, maybe the note about where OLM is installed should be put before the command to create the `CatalogSource` and a variable `$OLM_NAMESPACE` should be introduced to refer to that namespace?